MapboxCoreNavigation/RouteController.swift

```diff
@@ -288,7 +288,13 @@ open class RouteController: NSObject {
         guard let stepCoordinates = routeProgress.currentLegProgress.currentStep.coordinates else { return nil }
         guard let snappedCoordinate = closestCoordinate(on: stepCoordinates, to: location.coordinate) else { return location }
         
-        let nearByCoordinates = routeProgress.currentLegProgress.nearbyCoordinates
+        var nearByCoordinates = routeProgress.currentLegProgress.currentAndUpcomgingStepCoordinates
```

Comment:
Maybe this should remain `nearbyCoordinates`? Although I cant think of any reason to know about the previous step coords

Reply:
At a glance, I can’t think of any reason it would make a difference whether we include the previous coordinates or not.